package.json

```diff
@@ -14,7 +14,7 @@
     "pnpm": ">=7.14.2"
   },
   "scripts": {
-    "build": "pnpm --filter=@graphprotocol/graph-cli build",
+    "build": "pnpm --filter=@graphprotocol/graph-* build",
```

Comment:
@saihaj I guess we can just do without filter? this way it runs on all monorepos packages that has `build` script?  (@enisdenjo the pnpm master, to confirm 😆 )

Reply:
I did try that but issue is the examples and tests are generated which would error since they do not have `graph-cli` built since all this is done concurrently